lldb/source/Commands/CommandObjectCommands.cpp

```diff
@@ -1788,12 +1792,13 @@ class CommandObjectCommandsScriptDelete : public CommandObjectParsed {
       return;
     }
     const char *leaf_cmd = command[num_args - 1].c_str();
-    llvm::Error llvm_error = container->RemoveUserSubcommand(leaf_cmd,
-                                            /* multiword not okay */ false);
+    llvm::Error llvm_error =
+        container->RemoveUserSubcommand(leaf_cmd,
+                                        /* multiword not okay */ false);
     if (llvm_error) {
-      result.AppendErrorWithFormat("could not delete command '%s': %s",
-                                   leaf_cmd, 
-                                   llvm::toString(std::move(llvm_error)).c_str());
+      result.AppendErrorWithFormat(
+          "could not delete command '%s': %s", leaf_cmd,
+          llvm::toString(std::move(llvm_error)).c_str());
```

Comment:
revert non related whitespace change

Reply:
No, it is actually required by the formatter reported by github: 
https://github.com/llvm/llvm-project/actions/runs/7789051206/job/21239805569?pr=80375